plugins/t/check_smtp.t

```diff
@@ -55,6 +57,14 @@ SKIP: {
 	# SSL connection for STARTTLS
 	$res = NPTest->testCmd( "./check_smtp -H $host_tcp_smtp_starttls -p 25 -S" );
 	is ($res->return_code, 0, "OK, with STARTTLS" );
+
+	$res = NPTest->testCmd( "./check_smtp -H $host_tcp_smtp_starttls -p 25 -S -D 14" );
```

Comment:
Could you use `--starttls` here instead of `-S`. Would be more obvious in my eyes

Reply:
Agreed. Then, let's leave them for now.